extensions/amp-render/1.0/amp-render.js

```diff
@@ -246,9 +247,38 @@ export class AmpRender extends BaseElement {
       this.mutateProps(
         dict({
           'render': (data) => {
+            if (this.element.getAttribute('binding') === 'no') {
+              return templates
+                .renderTemplateAsString(dev().assertElement(template), data)
+                .then((html) => dict({'__html': html}));
+            }
+
+            let element;
             return templates
-              .renderTemplateAsString(dev().assertElement(template), data)
-              .then((html) => dict({'__html': html}));
+              .renderTemplate(dev().assertElement(template), data)
+              .then((el) => {
+                element = el;
+                return Services.bindForDocOrNull(this.element);
+              })
+              .then((bind) => {
```

Comment:
You'd probably also want to check if `bind` is `null` when unavailable, and return a simple template string in that case. 

For simplicity, `Service.bindForDocOrNull` should probably be the first promise to resolve.

Reply:
Done.